tests/common/test_cgroupapi.py

```diff
@@ -727,11 +752,14 @@ def mock_popen(*args, **kwargs):
                         stdout=stdout,
                         stderr=stderr)
 
-                    extension_calls = [args[0] for (args, _) in popen_patch.call_args_list if "echo 'success'" in args[0]]
+                    extension_calls = [args[0] for (args, _) in popen_patch.call_args_list if
+                                       "echo 'success'" in args[0]]
 
                     self.assertEquals(2, len(extension_calls), "The extension should have been invoked exactly twice")
-                    self.assertIn("systemd-run --unit=Microsoft.Compute.TestExtension_1.2.3", extension_calls[0], "The first call to the extension should have used systemd")
-                    self.assertEquals("echo 'success'", extension_calls[1], "The second call to the extension should not have used systemd")
+                    self.assertIn("systemd-run --unit=Microsoft.Compute.TestExtension_1.2.3", extension_calls[0],
+                                  "The first call to the extension should have used systemd")
+                    self.assertEquals("echo 'success'", extension_calls[1],
+                                      "The second call to the extension should not have used systemd")
```

Comment:
I spent several hours trying to figure out why the breakpoint in L755 was never hit... turns out we had a bug checked in where the indentation was such that the asserts were inside of the context manager, instead of outside it.

This is a nasty bug since it's so easy to miss. Could linters figure this one out? Thoughts, @kevinclark19a?

Reply:
Ah. Interesting. I'm not sure a linter could actually help with this... I don't see how it would differentiate between the legitimate caught raise statement presiding (presumably) inside of the `start_extension_command` call, and the erroneously caught one inside the `assertEquals` call. As far as the AST is concerned, they're basically identical. 

Still, there might be some way of using attributes to let the linter know what we want caught and what we want to error out... I'll have to look into it more.

Reply:
can you point out the change in indentation? I'm having a hard time finding it in the diff

Reply:
I thought I was blind.. turns out I commented on the wrong method. The content of the comment is correct, but the method in question is `test_start_extension_command_should_not_use_fallback_option_if_extension_fails` instead of this one.

Reply:
Added a comment on the right method.